src/modules/deleteTask.js

```diff
@@ -1,16 +1,13 @@
 /* eslint-disable linebreak-style */
 /* eslint-disable import/no-cycle */
 /* eslint-disable no-plusplus */
-import saveStorage from './saveStorage.js';
 
-const deleteTask = (itemIndex) => {
-  const storedStuff = JSON.parse(localStorage.getItem('todo-list'));
-
-  const newList = storedStuff.filter((i) => storedStuff.indexOf(i) !== itemIndex);
+const deleteTask = (itemIndex, stuff) => {
+  const newList = stuff.filter((i) => stuff.indexOf(i) !== itemIndex);
   for (let i = 0; i <= newList.length - 1; i++) {
```

Comment:
- [ ] For particular this disabling eslint, I would recommend to maintain the code standard instead of disabling. You can fix this by using `i += 1` on line `7` instead of `i++`.